src/planner/binder/statement/bind_create.cpp

```diff
@@ -615,6 +618,26 @@ BoundStatement Binder::Bind(CreateStatement &stmt) {
 		}
 		break;
 	}
+	case CatalogType::DATABASE_ENTRY: {
+		// not supported in DuckDB but allow replacement scan
+		auto &base = (CreateDatabaseInfo &)*stmt.info;
+		string extension_name = base.extension_name;
+		string database_name = base.name;
+		auto &config = DBConfig::GetConfig(context);
+		if (!config.create_database_extensions.empty()) {
```

Comment:
We want the extension to decide if it will honor the create database request.  Can achieve this by checking if the result.plan is populated after the `break` (which means that an extension bound a table function to do the create database).

i.e
```
for(auto &extension ...) {
...
if(create_database_function_ref) {
    ...
    break;
}
}
if(!result.plan) throw NotImpemented(...);
```

Reply:
makes sense. I applied this suggested code change. thanks!